documentation/_puppetdb_nav.html

```diff
@@ -60,6 +60,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/query.html">Query Structure</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/operators.html">Available Operators</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/paging.html">Query Paging</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/root.html">Root Endpoint (Experimental)</a></li>
```

Comment:
Would using the route `/pdb/query/v4` be crazy?
If so what about `/pdb/query/v4/context`  or `/pdb/query/v4/from` to match it with the operator?